### PR TITLE
Feature/convert to onnx and openvino scripts

### PR DESCRIPTION
**Pull request by phinik**
2 Scripts to convert a PyTorch model to ONNX and from ONNX to OpenVino IR.

TODO:

- [X] Update repo version number

### CODE REVIEW

**Comment by Flova** [comment]
Ping @phinik ? The scripts lgtm maybe we should also open a upstream pr at pytorchyolo.

**Comment by jaagut** [comment]
@Flova why ping? Have you tested the scripts? I believe this is ready for review/merge.

**Comment by jaagut** [comment]
We can ignore the failing CI for now, as it is not yet fully working...

**Comment by phinik** [comment]
> @Flova Have you tested the scripts? I believe this is ready for review/merge.

pls, don't merge yet.

**Comment by phinik** [comment]
> > @Flova Have you tested the scripts? I believe this is ready for review/merge.
> 
> pls, don't merge yet.

Can be merged

**Comment by phinik** [comment]
> Ping @phinik ? The scripts lgtm maybe we should also open a upstream pr at pytorchyolo.

Pytorchyolo doesn't have scripts yet, right?

**Comment by Flova** [comment]
That is correct. I would suggest adding it.

File: pyproject.toml [hunk 1]
```diff
@@ -34,3 +34,5 @@ build-backend = "poetry.core.masonry.api"
 yoeo-detect = "yoeo.detect:run"
 yoeo-train = "yoeo.train:run"
 yoeo-test = "yoeo.test:run"
+yoeo-to-onnx = "scripts.convertPyTorchModelToONNX:run"
```
Comment (Flova) [inline pyproject.toml:37]: Now this is located in the scripts package and not the YOEO python package.
This means that when installing YOEO as a python package outside a venv, the scripts might be missing or in a new `scripts` package which would be not so clean.

Comment (Flova) [reply pyproject.toml:37]: We should think about reworking this structure

**Review by Flova — APPROVED** [review]

**Pull request merged**